tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -165,7 +191,17 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
        |override def clone(): Object
        |override def finalize(): Unit
        |""".stripMargin,
-    includeDetail = false
+    includeDetail = false,
+    topLines = if (scalaVersion.startsWith("3")) Some(5) else None,
```

Comment:
What is being shown without `Some(5)`?

Reply:
For example, if we set `Some(7)`, we get the following result,

```
    """|override def aaa: Int
       |override def bbb: Int
       |override def equals(x$0: Any): Boolean
       |override def hashCode: Int
       |override def toString: String
       |<error>: Null
       |Abstract empty
       |""".stripMargin
```

and `<error>: Null`  and `Abstract empty` come from compiler completions.

Probably it's better to suppress fromCompiler completions in some patterns, but we should do that in other PR (because this PR is already large). Therefore, restrict the number of completions here and hide uninteresting items.
Anyway, I'll leave a comment about why I'm doing this.

Reply:
Ok sure, we should follow up and maybe filter error complerions?

Reply:
We don't neet it anymore thanks to https://github.com/scalameta/metals/pull/3897#discussion_r875491408 :tada: